code/game/objects/items/weapons/material/knives.dm

```diff
@@ -79,6 +79,6 @@
 	name = "butcher's cleaver"
 	icon = 'icons/obj/kitchen.dmi'
 	icon_state = "butch"
-	desc = "A huge thing used for chopping and chopping up meat. This includes clowns and clown-by-products."

```

Comment:
I actually think this one is okey, it doesnt really reference "THe clown" just clowns and doesnt everyone want to chop up clowns?